custom/icds_reports/reports/service_delivery_report.py

```diff
@@ -120,24 +120,38 @@ def _location_name():
         return headers
 
     def get_excel_data(self):
+        total_sum_row = [''] * self.config['aggregation_level'] + [0] * (
+            len(self.headers_and_calculation) - self.config['aggregation_level']
+        )
```

Comment:
nit: this line took me just long enough to figure out that it may be worth a quick comment

Reply:
sure.